CorsixTH/Lua/persistance.lua

```diff
@@ -38,13 +37,8 @@ if _G._VERSION == "Lua 5.2" or _G._VERSION == "Lua 5.3" then
     end
     return val
   end
-
-  th_getupvalue = function(f, n)
-    return debug.getupvalue(f, n + 1)
-  end
 else
   th_getfenv = debug.getfenv;
-  th_getupvalue = debug.getupvalue;
```

Comment:
this one is tricky to delete; it looks like it is used for debugging, so the function may get called from something outside this program (which luacheck probably doesn't understand), better keep this